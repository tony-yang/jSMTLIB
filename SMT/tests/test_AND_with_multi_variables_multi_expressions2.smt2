(set-logic QF_LRA)
(set-info :source | SMT-COMP'06 organizers |)
(set-info :smt-lib-version 2.0)
(set-info :category "check")
(set-info :status sat)
(set-info :notes |This benchmark is designed to check if the DP supports bignumbers.|)
(declare-fun x1 () Real)
(declare-fun x2 () Real)
(assert (and (and (> (+ (* 3 x1) (* 2 x2)) 10) (< (+ (* 6 x1) (* 8 x2)) 60)) (= (+ (* 7 x1) (* (- 9) x2)) 100)))
(check-sat)
(exit)
