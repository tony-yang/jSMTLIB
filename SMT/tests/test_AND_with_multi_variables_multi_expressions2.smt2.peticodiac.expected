# set_logic: QF_LRA
# :source: | SMT-COMP'06 organizers |
# :smt-lib-version: 2.0
# :category: "check"
# :status: sat
# :notes: |This benchmark is designed to check if the DP supports bignumbers.|
p cnf 2 3
c 3.0 2.0
b 2 >:10 NO_BOUND
c 6.0 8.0
b 3 NO_BOUND <:60
c 7.0 -9.0
b 4 =:100 =:100
